components/layout_2020/replaced.rs

```diff
@@ -71,11 +71,8 @@ impl ReplacedContent {
     }
 
     fn flow_relative_intrinsic_size(&self, style: &ComputedValues) -> Vec2<Option<Length>> {
-        let intrinsic_size = physical::Vec2 {
-            x: self.intrinsic.width,
-            y: self.intrinsic.height,
-        };
-        intrinsic_size.size_to_flow_relative(style.writing_mode)
+        let intrinsic_size = PhysicalSize::new(self.intrinsic.width, self.intrinsic.height);
```

Comment:
Passing two arguments of the same type where order is significant instead of writing out named fields in a struct literal seems like a regression, but oh well.

Reply:
I totally agree. This seems to be one of the sharper edges of euclid and I wish there was a way to provide named arguments or to instantiate the struct right here. It seems that it is impossible though due to it having private fields.